lib/install/management/vmgroup.go

```diff
@@ -34,36 +35,78 @@ func (d *Dispatcher) createVMGroup(conf *config.VirtualContainerHostConfigSpec)
 
 	d.op.Debugf("Creating DRS VM Group %q on %q", conf.VMGroupName, d.appliance.Cluster)
 
-	spec := &types.ClusterConfigSpecEx{
-		GroupSpec: []types.ClusterGroupSpec{
-			{
-				ArrayUpdateSpec: types.ArrayUpdateSpec{
-					Operation: types.ArrayUpdateOperationAdd,
-				},
-				Info: &types.ClusterVmGroup{
-					ClusterGroupInfo: types.ClusterGroupInfo{
-						Name: conf.VMGroupName,
+	_, err := tasks.WaitForResultAndRetryIf(d.op, func(op context.Context) (tasks.Task, error) {
+		var containers []types.ManagedObjectReference
+		rp, err := d.getComputeResource(nil, conf)
+		if err != nil {
+			return nil, err
+		}
+
+		d.parentResourcepool = rp
+
+		children, err := d.parentResourcepool.GetChildrenVMs(d.op, d.parentResourcepool.Session)
```

Comment:
You could use this [func](https://github.com/vmware/vic/blob/master/lib/install/management/inventory.go#L171) that would return you an array of child moRefs that are containers -- more efficient than getting children and calling `d.isContainerVM` on each vm